jenkins-pipelines/nemesis/longevity-5gb-1h-RandomInterruptionNetworkMonkey-aws.jenkinsfile

```diff
@@ -7,6 +7,6 @@ longevityPipeline(
     backend: 'aws',
     region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: 'test-cases/nemesis/longevity-5gb-1h-RandomInterruptionNetworkMonkey.yaml',
-    ip_ssh_connections: 'public'
```

Comment:
same as above, we need this pipeline as well

Reply:
Fixed